pages/home.tsx

```diff
@@ -1,222 +1,34 @@
-import InputField from '../components/InputField'
```

Comment:
The re-use of this page name `home` for something else, will break the history. I think it's better to add the splash page under another name; then later we can come back to renaming files.